apps/src/templates/instructions/inline-audio.module.scss

```diff
@@ -0,0 +1,5 @@
+@import "../../mixins.scss";
+
+.inlineAudioButton {
+  @include remove-button-styles;
```

Comment:
do these buttons have any sort of hover indicator? In general we want a hover indication, preferably a color change (see [this thread](https://codedotorg.slack.com/archives/C046F4VNQ7Q/p1669935297409089) for details).

Reply:
Good call! To clarify - this button does change color when mousing over, but should it also change to the same color when selected via keyboard navigation? I don't think it does that currently.

Reply:
I think it should also change color on keyboard navigation

Reply:
Gotcha - looks like this component uses some custom mouseOn/mouseOff logic to apply hover styles (as opposed to using :hover). I might split this into a separate fast-follow in order to rework this to use :hover if that's cool?

Reply:
that seems reasonable